src/localization/templates/sr-Latn-RS.json

```diff
@@ -87,6 +87,7 @@
   "tokenKind_lessThanEqualTo": "operator „manje od ili jednako“ ('<=')",
   "tokenKind_minus": "minus <'-'>",
   "tokenKind_notEqual": "operator „nije jednako“ ('<>')",
+  "tokenKind_nullCoalescingOperator": "TODO",
```

Comment:
Are we supposed to update these template files? 
Ideally the loader would substitute in the default/en-US message if the localized message wasn't found.

Reply:
Talked offline. Waiting for the update localization files to be dropped.